ci/docker/install/ubuntu_arm.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Please rename file to match our filenaming style.

Reply:
What exactly the name you are suggesting in this case?

Reply:
Nevermind, this is fine :)